src/main/scala/com/microsoft/ml/spark/lightgbm/LightGBMClassifier.scala

```diff
@@ -86,16 +86,27 @@ trait HasLeafPredictionCol extends Params {
   def setLeafPredictionCol(value: String): this.type = set(leafPredictionCol, value)
 }
 
+trait HasFeaturesShapCol extends Params {
+  val featuresShapCol = new Param[String](this, "featuresShapCol",
+    "features' shap value column name that means their contribution for prediction")
```

Comment:
Maybe change the description to:
```
Output SHAP vector column name after prediction containing the feature contribution values
```

Reply:
@imatiach-msft It is changed to your suggestion